torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -3291,6 +3303,54 @@ def _check_rebuild_full_params(self, param: FlatParameter):
                     )
             eod.param_order.append(param_index)
 
+    def register_comm_hook(self, state: object, hook: callable):
+        r"""
```

Comment:
could you please build the docs and check if this is rendered properly?

Reply:
I've checked docs build without issues, will verify how it is rendered in PR's docs